nx-X11/programs/Xserver/composite/compwindow.c

```diff
@@ -524,6 +588,7 @@ compCopyWindow (WindowPtr pWin, DDXPointRec ptOldOrg, RegionPtr prgnSrc)
 		}
 		FreeScratchGC (pGC);
 	    }
+            REGION_UNINIT (pWin->drawable.pScreen, &rgnDst);
```

Comment:
Also in https://github.com/ArcticaProject/nx-libs/pull/125/commits/d64eabf3d4b233bdbc10f40fa3df653d489101fc.